lib/candidates.js

```diff
@@ -102,6 +102,8 @@ exports.editCandidature = async (req, res) => {
     delete req.body.status;
     delete req.body.user_id;
 
+    req.body.body_name = req.user.bodies.find((body) => body.id === req.body.body_id).name;
```

Comment:
Can there be cases when there's no body? It'll crash there

Reply:
Nope. This is the same line of code we use when the candidature is created btw